dotcom-rendering/cloudformation.yml

```diff
@@ -247,12 +247,12 @@ Resources:
           usermod -a -G frontend aws-kinesis-agent-user
           cd /home/dotcom-rendering
 
-          aws --region eu-west-1 s3 cp s3://aws-frontend-artifacts/frontend/${Stage}/${App}/dist/${App}.zip ./
+          aws --region eu-west-1 s3 cp s3://aws-frontend-artifacts/frontend/${Stage}/${App}/${App}.zip ./
           unzip -q ${App}.zip -d ${App}
 
           chown -R dotcom-rendering:frontend ${App}
 
-          cd ${App}/dotcom-rendering
```

Comment:
We're a lot more specific when creating the ZIP bundle now and everything required to run the DCR server is in the root folder of the bundle now.

Previously the bundle contained the entire repository, including common-rendering, which is why we'd have to cd into the dotcom-rendering subfolder.